src/main/java/com/akukhtin/springbootapplication/controller/RegistrationController.java

```diff
@@ -0,0 +1,17 @@
+package com.akukhtin.springbootapplication.controller;
+
+import com.akukhtin.springbootapplication.dto.UserDto;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class RegistrationController {
+    @RequestMapping(value = "/user/registration", method = RequestMethod.GET)
```

Comment:
```suggestion
    @RequestMapping(value = "/registration", method = RequestMethod.GET)
```
зазвичай сторінка реєстрації як і сторінка логіну пишется від рута